### DISCUSSION TITLE
my t-pot doesn't show anything just the time

### DISCUSSION THREAD

**Discussion by marthexmit** [telekom-security/tpotce#1432] [Q&A]
my t-pot doesn't show anything just the time
![tpot](https://github.com/telekom-security/tpotce/assets/72632039/12fe6fac-bafa-45e0-a5ba-c762c60f96b3)

**Comment by PLACIDEM** [comment]
I ecountered  same issue  when  try to run  Tpot  from the same VM  machine where  it is installed  by (https://localhost:64297) and I had the same  screenshot  above. however when I tried access tpot  from my Host machine  accessing VM, it works.
